marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/mapping/removeAttachment/runMappingStepWithAndWithoutAttachment.sjs

```diff
@@ -0,0 +1,37 @@
+const DataHubSingleton = require("/data-hub/5/datahub-singleton.sjs");
+const test = require("/test/test-helper.xqy");
+const hubTest = require("/test/data-hub-test-helper.sjs");
+const datahub = DataHubSingleton.instance();
+let assertions = [];
+
+const mappingStep = hubTest.getRecord("/steps/mapping/mappingStep1.step.json").document;
+
+assertions = assertions.concat(test.assertEqual(false, mappingStep.attachSourceDocument));
+
+let content = ['/content/customerInfoWithoutAttachment.json'].map(uri => {
+  return {
+    uri: uri,
+    value: fn.head(xdmp.unquote('{"customerId": 100, "name": "Cynthia Waters"}'))
+  };
+});
+datahub.flow.runFlow('simpleMappingFlow', 'testJob', content, {outputFormat: 'json', mapping:{name:'mappingStep1'}}, 1);
```

Comment:
Note that using the flowApi function is simpler - "flow.runFlow" doesn't actually run a flow, it's really "process a batch of content".